lametro/templates/index/_meeting_details_next.html

```diff
@@ -34,16 +34,19 @@ <h3 class="h4">{{  meeting.link_html | safe }}</h3>
         </p>
 
         <!-- Supplementary links -->
-        {% if meeting.documents.all %}
-          <div class="row">
-            <div class="col-7">
-              <a class="btn btn-teal" id="pdf-download-link" target='_blank' href='#' aria-label="Get Agenda PDF - link opens in a new tab">
-                <i class='fa fa-fw fa-download' aria-hidden="true"></i>
-                Get Agenda PDF
-              </a>
+        {% with agenda_url=meeting.documents.all|find_agenda_url %}
```

Comment:
What's up with this change?

Reply:
This was part of [a fix in this other pr](https://github.com/Metro-Records/la-metro-councilmatic/pull/1218). I merged in main to keep the cms branch up to date (since learning my lesson with the a11y branches 😱).

But that fix was mostly to get the pdf agenda links working again, since they were previously empty.